src/backend/InvenTree/plugin/installer.py

```diff
@@ -108,8 +126,17 @@ def install_plugins_file():
         logger.warning('Plugin file %s does not exist', str(pf))
         return
 
+    cmd = [
+        'install',
+        '--disable-pip-version-check',
+        '--no-cache-dir',
```

Comment:
Q: Are you sure we do not want to use caching?

Reply:
No probably not - I have reverted this now